app/uk/gov/hmrc/debttransformationstub/utils/ifsrulesmasterspreadsheet/impl/IfsRulesMasterData.scala

```diff
@@ -63,19 +63,22 @@ final case class IfsRulesMasterData(
           )
       }
 
-    def useChargeReference(index: Int): Option[Boolean] =
-      Lookup2D.chargeReferenceAt(index).actual match {
-        case "N/A"        => None
-        case "Charge ref" => Some(true)
-        case "ASN"        => Some(false)
-        case "VRN"        => Some(false)
-        case "UTR"        => Some(false)
-        case unknown =>
+    def useChargeReference(index: Int): Option[Boolean] = {
+      val chargeReference: String = Lookup2D.chargeReferenceAt(index).actual
+
+      chargeReference.toLowerCase match {
+        case "n/a"        => None
+        case "charge ref" => Some(true)
+        case "asn"        => Some(false)
+        case "vrn"        => Some(false)
+        case "utr"        => Some(false)
+        case _ =>
           val rowDisplay: JsValue = Json.toJson(tableData.dataRowAt(index).values.map(_.actual))
           throw new IllegalArgumentException(
-            s"Cannot convert useChargeReference=${JsString(unknown)} to boolean; check the code. Row values: $rowDisplay"
+            s"Cannot convert useChargeReference=${JsString(chargeReference)} to boolean; check the code. Row values: $rowDisplay"
```

Comment:
Can we change this back to `unknown`

Reply:
I used `chargeReference` so that the string case is preserved, rather than use the lowercase `unknown` charge ref. Would you prefer I use that here?

Reply:
Ah I see, no that makes sense